src/modules/blockchainApplication/connection/components/ConnectionSummary/index.js

```diff
@@ -4,16 +4,78 @@ import { useTranslation } from 'react-i18next';
 import Dialog from '@theme/dialog/dialog';
 import grid from 'flexboxgrid/dist/flexboxgrid.css';
 import ValueAndLabel from 'src/modules/transaction/components/TransactionDetails/valueAndLabel';
-import { PrimaryButton, SecondaryButton } from 'src/theme/buttons';
+import { PrimaryButton, SecondaryButton, TertiaryButton } from 'src/theme/buttons';
 import { ACTIONS, EVENTS } from '@libs/wcm/constants/lifeCycle';
 import { addSearchParamsToUrl } from 'src/utils/searchParams';
 import { useEvents } from '@libs/wcm/hooks/useEvents';
 import { useSession } from '@libs/wcm/hooks/useSession';
 import classNames from 'classnames';
+import { useBlockchainApplicationMeta } from '@blockchainApplication/manage/hooks/queries/useBlockchainApplicationMeta';
+import { getLogo } from '@token/fungible/utils/helpers';
+import Icon from '@theme/Icon';
 import BlockchainAppDetailsHeader from '../../../explore/components/BlockchainAppDetailsHeader';
 import AccountsSelector from './AccountsSelector';
 import styles from './connectionSummary.css';
 
+function ChainListingItem({ app }) {
+  const { t } = useTranslation();
+  const { chainName, networkType, chainID, logo } = app;
+  const name = `${chainName}-${networkType}`;
+  const logoUrl = getLogo({ logo });
+
+  return (
+    <div className={styles.ChainListingItem}>
+      <img
+        className={styles.chainLogo}
+        height={40}
+        width={40}
+        src={logoUrl}
+        alt={`${chainName}-logo`}
+      />
+      <div className={styles.textContainer}>
+        <h4 className={styles.name}>{name}</h4>
+        <span className={styles.chainId}>{t('Chain ID: ') + chainID}</span>
+      </div>
+    </div>
+  );
+}
+
+function ChainListing({ chainIds }) {
+  const chainIDs = chainIds?.join(',');
+  const blockchainApplicationMeta = useBlockchainApplicationMeta({
+    config: { params: { chainID: chainIDs } },
+    options: { enabled: !!chainIDs?.length },
+  });
+
+  return (
+    <div className={styles.ChainListing}>
+      {blockchainApplicationMeta?.data?.data?.map((app) => (
+        <ChainListingItem key={app.chainID} app={app} />
+      ))}
+    </div>
+  );
+}
+
+function CollapsableRow({ label, children }) {
+  const [showChildren, toggleShowChildren] = useState(true);
+
+  return (
+    <div className={styles.CollapsableRow}>
+      <TertiaryButton
+        className={classNames(styles.rowHeaderButton, showChildren && styles.marginBottom)}
+        onClick={() => toggleShowChildren(!showChildren)}
+      >
+        <span>{label}</span>
+        <Icon
+          name="arrowRightInactive"
+          className={classNames(styles.arrowIcon, showChildren && styles.showArrowDown)}
```

Comment:
I think its more readable this way

```suggestion
          className={classNames(styles.arrowIcon, { styles.showArrowDown: showChildren })}
```